### ISSUE TITLE
Glass doesn't work

### ISSUE THREAD

**Issue by realGameknight999** [Wyn-Price/SecretRooms#14]
The hidden glass in the 1.14.4 version of the mod won't work. I place it, and it's just normal glass, I even tried changing the block data, but that didn't work, please fix this ASAP.

**Comment by Wyn-Price** [comment]
Can you send screenshots, along with the latest.log

**Comment by realGameknight999** [comment]
![2019-11-10_16 58 38](https://user-images.githubusercontent.com/47459404/68551566-38de4280-03dc-11ea-9f5f-f814494a032f.png)

**Comment by realGameknight999** [comment]
![2019-11-10_16 58 44](https://user-images.githubusercontent.com/47459404/68551581-6b883b00-03dc-11ea-90a8-3f165332a097.png)

**Comment by realGameknight999** [comment]
![2019-11-10_16 58 44](https://user-images.githubusercontent.com/47459404/68551593-8955a000-03dc-11ea-9c5c-bb7891041831.png)

**Comment by Wyn-Price** [comment]
And if not all of the properties are true, does that still cause issues? 

And if you could also send over the latest.log

**Comment by realGameknight999** [comment]
![2019-11-10_16 58 59](https://user-images.githubusercontent.com/47459404/68551599-94103500-03dc-11ea-99ca-5a8ec8b4b6b2.png)

**Comment by realGameknight999** [comment]
I tried doing that with /data modify

**Comment by Wyn-Price** [comment]
When you place it normally (with the item) it shouldn't be that all states are true

**Comment by realGameknight999** [comment]
[latest.log](https://github.com/Wyn-Price/SecretRooms/files/3829098/latest.log)

**Comment by realGameknight999** [comment]
Well, it was like that

**Comment by Wyn-Price** [comment]
Can you post a screenshot of the when the glass is placed normally?

**Comment by realGameknight999** [comment]
![2019-11-10_17 12 33](https://user-images.githubusercontent.com/47459404/68551683-6d063300-03dd-11ea-92c2-713b4cc50f16.png)

**Comment by realGameknight999** [comment]
Am I using an outdated Forge version?

**Comment by realGameknight999** [comment]
Did you fix it?

**Comment by Wyn-Price** [comment]
Nothing you've posted so far is an actual issue. All the sides of that class are true, so they're going to render as glass. Please right click one of the glass sides with an empty hand. 

When you place the glass block down, does it get set down like you've shown (all sides as true)

**Comment by Wyn-Price** [comment]
Also, try with only secretrooms mod, not other mods. It could be a conflict

**Comment by fuzzyweapon** [comment]
Right, I can confirm this works (using an empty hand) on latest with 1.14.4.  I've noticed a lot of players just don't know how to use it.  I tested it with lots of other mods, but Wyn-Price's suggestion is the best way to isolate issues and reduce erroneous variables in your testing.

**Comment by Wyn-Price** [comment]
I've been meaning to write up about the changes and features in 1.14, but I've never managed to find the time to. I'll try and do it all tomorrow so issues like these don't happen.

**Comment by realGameknight999** [comment]
Ok, thank you for telling me how it works, turns out it’s not broken and it
works

**Comment by Wyn-Price** [comment]
Does this still occur
